File: notify/docs/sending-texts/send_multi.go
templateID := "..."

texts := []notify.Text{
	notify.Text{
		PhoneNumber: "...",
		Reference:   "...",
		TemplateData: notify.TemplateData{
			"name": "Kim",
		},
	},
	notify.Text{
		PhoneNumber: "...",
		Reference:   "...",
		TemplateData: notify.TemplateData{
			"name": "Alex",
		},
	},
}

res, err := client.SendTextMulti(ctx, templateID, texts)
if err != nil {
	// ...
}

File: notify/docs/sending-texts/send_multi.js
const templateId = "...";

const texts = [
  {
    phoneNumber,
    reference,
    templateData
  },
  {
    phoneNumber,
    reference,
    templateData
  }
];

client
  .sendTextMulti({ templateId, texts })
  .then(response => console.log(response))
  .catch(err => console.error(err));
